indexing-service/src/main/java/io/druid/indexing/worker/Worker.java

```diff
@@ -44,7 +44,7 @@ public Worker(
       @JsonProperty("version") String version
   )
   {
-    this.scheme = scheme;
+    this.scheme = scheme == null ? "http" : scheme;
```

Comment:
Add a comment that this is needed for backwards compatibility with older workers (pre-#4270)  and I think this is good to go.

Reply:
added